src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -25,6 +25,10 @@ const Body = BaseForm.extend(Object.assign(
 
     initialize() {
       BaseForm.prototype.initialize.apply(this, arguments);
+      // 'hasSavingState' would be true when the auth key is custom_app.
+      // Setting it to false when auth key is okta_verify.
+      // So that 'o-form-saving' css class is not added while polling and checkbox remains enabled. 
+      this.hasSavingState = !this.isOV();
```

Comment:
should this then be
```
if (this.isOV()) {
  this.hasSavingState = false;
}
```
otherwise we would be overriding the value in flows where this is not OV, without knowing what it was before

Also, should we also check if `autoChallenge` is present? Otherwise we don't need to change this right? so maybe something like
```
if (this.isOVWithAutoChallenge() {
  this.hasSavingState = false;
}
```

Reply:
This view shows 'Get a push notification' view that has this button disabled. Going forward the view will also have the checkbox. In this specific case I dont think checking for `isOVWithAutoChallenge` is required. My understanding is there cannot be any other ov factor in this page

Reply:
the same view is used when the key is 'okta_verify' or 'custom_app'. Currently `hasSavingState` is true always. With this change we are forcing `hasSavingState` to be true only when key is other than 'okta_verify'. Its same as having it under `if` condition as suggested. If you still feel otherwise, can change it.